content/applications/marketing/sms_marketing/pricing/pricing_and_faq.rst

```diff
@@ -31,106 +30,81 @@ Limit per SMS: 70 characters.
-
+   :align: center
+   :alt: Using Odoo IAP to recharge credits for SMS Marketing in Odoo settings.
 
 .. tip::
-   If you are on Odoo Online and have the Enterprise version, you benefit from free trial credits to
-   test the feature.
-
+   If you are on Odoo Online (SAAS) and have the Enterprise version, you benefit from free trial
+   credits to test the feature.
 
 More common questions
 =====================
 
-#. **Is there an expiration time for my credits?**
-
-   No, credits do not expire.
-
-
-#. **Can I send an SMS to a phone number (which is not a mobile phone) because I see the icon in
-   front of the field “phone”?**
-
-   Only if that phone number supports SMS (e.g. SIP phones).
-
-
-#. **Do I receive an invoice to buy my credits?**
-
-   Yes.
-
-
-#. **Can the recipient answer to me?**
-
-   No, it is not possible to reply to the SMS.
-
-
-#. **What happens if I send multiple SMS but I do not have enough credits to send them all?**
-
-   The whole transaction is counted as a single one, so no SMS will be sent until you have enough
-   credits to send them all.
-
-
-#. **Do I have a history of the sent SMSs?**
-
-   Yes, the SMS is logged as a note under the chatter of the corresponding record, and a complete
-   history of the SMSs sent is available on :ref:`developer mode <developer-mode>` under
-   :menuselection:`Technical --> SMS`.
-
-
-#. **Can I send as many SMSs I want at once?**
-
-   Yes, if you have enough credits.
-
-
-#. **If I have a number that does not exist in the list of recipients, will I lose credits?**
-
-   Not if the phone number is in the wrong format (e.g. too many digits). Otherwise, if the SMS is
-   sent to the wrong person or to a fake number, the credit will be lost.
+#. | **Is there an expiration time for my credits?**
+   | No, credits do not expire.
 
+#. | **Can I send an SMS to a phone number (which is not a mobile phone) because I see the icon in
+     front of the field “phone”?**
+   | Only if that phone number supports SMS (e.g. SIP phones).
 
-#. **What happens if I send my SMS to a paying number (e.g.: a contest to win a ticket for a festival)?**
+#. | **Do I receive an invoice to buy my credits?**
+   | Yes.
 
-   The SMS will not be delivered to that kind of number, so you won’t be charged.
+#. | **Can the recipient answer to me?**
+   | No, it is not possible to reply to the SMS.
 
+#. | **What happens if I send multiple SMS but I do not have enough credits to send them all?**
+   | The whole transaction is counted as a single one, so no SMS will be sent until you have enough
+     credits to send them all.
 
-#. **Can I identify the numbers that do not exist when I send several SMSs?**
+#. | **Do I have a history of the sent SMSs?**
+   | Yes, the SMS is logged as a note under the chatter of the corresponding record. A complete
+     history of SMSs cancelled / queued up / facing delivery errors is available on :ref:`developer
+     mode <developer-mode>` under.
```

Comment:
This seems like an incomplete thought/sentence, am I wrong? 

Once in Developer Mode, this information would be found under...which section, exactly? 

Also, the initial person who created this PR, had an issue with this question, and suggested the answer reflect the following: "SMS menu does not display SMSs that have been sent. Similar to the Technical ‣ Emails menu, the SMS menu only shows SMS's that are cancelled, queued up, or encountering delivery errors."

I was unable to find any history of SMSs in the chatter anywhere. I even checked the Contact detail form of a person who received a message, and it wasn't there. I also checked the Chatter of a specific SMS campaign (from the SMS Marketing dashboard), and it wasn't there, either. However, there's a blue banner on the detail form of each message, indicating how many have been sent/bounced back. When clicked, it reveals all the numbers that have successfully received it/didn't receive it. 

So, all that to say, I think the easiest way to answer this is to say something like:

"A history of sent SMSs, along with all pertinent information related to its sent contacts (and the message itself), can be found in the 'Sent' column of the main SMS Marketing dashboard (while in Kanban view). 

For more detailed information, select a desired SMS from the main dashboard (in Kanban view), and click on either link in the blue banner above the SMS detail form to learn more."

However, if you *did* find that SMS communications *are* present in the Chatter, and I just wasn't looking in the right place, could you please elaborate on how to locate such information? Because, at the moment, I'm having a difficult time trying to ascertain that information for myself. 

Regardless, the above suggested answer to this question should suffice.